src/RegexColumnizer/RegexColumnizer.cs

```diff
@@ -71,6 +71,16 @@ public IColumnizedLogLine SplitLine(ILogLineColumnizerCallback callback, ILogLin
                         Parent = logLine,
                         FullValue = line.FullLine
                     };
+                    
+                    //Fill other columns with empty string to avoid null pointer exceptions in unexpected places
+                    for (int i = columns.Length - 2; i >= 0; i--)

```

Comment:
The loop is reversed because the other loops in this file are also reversed and I was trying to use the same code style as previously used. That is also where the -2 comes from. I updated the loop to be in assenting order. 

A foreach loop is not possible here (IMHO) since the array is modified in the loop.